ghscrap/scrapper.py

```diff
@@ -16,25 +17,19 @@ def kitchen(params):
 
 def user_search_filter(filterkey):
     """Filter for user search pages.
-
```

Comment:
Add all these deleted blank lines.
They are according to coding style guide [PEP8](https://www.python.org/dev/peps/pep-0008/)